lib/db/upgrade.php

```diff
@@ -3907,6 +3907,40 @@ function xmldb_main_upgrade($oldversion) {
         upgrade_main_savepoint(true, 2022020200.01);
     }
 
+    // Unique indexes for question_set_references.
```

Comment:
add this to the end of the file, following the pattern like the previous ones, like a unique savepoint and oldversion which greater than the current version of core

Reply:
Added in date order.